docs/operations/rwx.md

```diff
@@ -1,136 +1,431 @@
-```
-
-Features of the NFS volume can also be examined in the Ondat UI.
-Under `Volumes`, the `Attachment` column shows an `nfs` tag. The
-`Volume Details` section provides information about the NFS Volume, such
-as the service endpoint and the node on which the underlying Volume is
-attached.
-
-## Changing the Squash mode
-
-As part of the 2.8 release, a change was made so that users can change the
-squash mode that is used with the RWX shares.
-Historically, all shares were exported with a `Squash = All` mode of operation.
-This was requested by most customers as the idea of identity in a container
-based deployment is very abstract.
-There is now a label that can applied to make this tunable, users can now add
-`storageos.com/nfs-squash = root|rootuid|all|none` to their volumes (where
-all is still the default behaviour still) to adjust this setting.
-
-As an example, the above PVC definition can be labelled as:
-
-```yaml
-apiVersion: v1
-kind: PersistentVolumeClaim
-metadata:
-  name: pvc-rwx
-  labels:
-    storageos.com/nfs-squash: rootuid
-spec:
-  storageClassName: storageos
-  accessModes:
-    - ReadWriteMany
-  resources:
-    requests:
-      storage: 5Gi
-```
-
-This will squash only the root user to `root` leaving other UID operations intact.
+## Overview
+
+Ondat support [`ReadWriteMany`](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes) (RWX) persistent volumes. A RWX volume can be used simultaneously by different deployments conducting read and write operations in the same namespace.
+
+> ⚠️ RWX volume provisioning is available in the free Ondat Community Edition. To get the Community Edition licence, register your cluster through [Ondat's SaaS platform](https://portal.ondat.io/) and generate a licence so that it can be applied to your cluster. For more information on licences, review the [Ondat pricing](https://www.ondat.io/pricing) page.
+
+> 💡 For more information on the Ondat Files feature, review the  [Ondat Files](https://github.com/ondat/documentation/blob/main/docs/concepts/encryption)  feature page.
+
+### Example - Use the RWX Access Mode Through a `PersistentVolumeClaim` Definition
+
+The following guidance will demonstrate how to use RWX volumes through a  `PersistentVolumeClaim`  (PVC) definition.
+
+- The instructions will allow you to use the RWX access mode on a PVC that will be mounted onto a  `Deployment`  resource in the  `ondat-files`  namespace.
+
+1. Create a namespace called `ondat-files` where the encrypted volume and  `Deployment`  will reside.
+
+ ```bash
+ # Create namespace called "ondat-files".
+ cat <<EOF | kubectl create --filename -
+ apiVersion: v1
+ kind: Namespace
+ metadata:
+   name: ondat-files
+   labels:
+     name: ondat-files
+ EOF
+ ```
+
+1. Create a custom `PersistentVolumeClaim` named `ondat-files` and ensure that you add the following `accessMode` >> `ReadWriteMany`  added to the manifest.
```

Comment:
...and ensure that you add the following `accessMode` >> `ReadWriteMany` to the manifest.

Reply:
fixed